operation/manifest.go

```diff
@@ -81,6 +81,36 @@ type AppManifestService struct {
 	Parameters  map[string]interface{} `yaml:"parameters,omitempty"`
 }
 
+func (ams *AppManifestService) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var raw interface{}
+	if err := unmarshal(&raw); err != nil {
+		return err
+	}
+	switch v := raw.(type) {
+	case string:
+		ams.Name = v
+	case map[interface{}]interface{}:
```

Comment:
When we try unmarshaling from raw yaml type directly it needs map[string{}]interface{} , hence not getting into this case due to map[interface{}]interface{} type check
Please change to map[string]interface{}

![image](https://github.com/cloudfoundry-community/go-cfclient/assets/33289859/24038cbd-35b2-49df-b322-7f5ec871c2f2)